Plugins/Creature/Creature.cpp

```diff
@@ -101,6 +101,27 @@ NWNX_EXPORT ArgumentStack RemoveFeat(ArgumentStack&& args)
     return {};
 }
 
+NWNX_EXPORT ArgumentStack RemoveFeatByLevel(ArgumentStack&& args)
+{
+    if (auto *pCreature = Utils::PopCreature(args))
+    {
+        const auto feat  = args.extract<int32_t>();
+          ASSERT_OR_THROW(feat >= Constants::Feat::MIN);
+          ASSERT_OR_THROW(feat <= Constants::Feat::MAX);
+        const auto level = args.extract<int32_t>();
+          ASSERT_OR_THROW(level >= 1);
+          ASSERT_OR_THROW(level <= Globals::AppManager()->m_pServerExoApp->GetServerInfo()->m_JoiningRestrictions.nMaxLevel);
+
+        if (level > 0 && level <= pCreature->m_pStats->m_lstLevelStats.num)
```

Comment:
nit: level is always >=1,as we have asserted just above.

Reply:
I kept it consistent with AddFeatByLevel. I can update both occurrences if you want me to.